src/vivliostyle/viewer.js

```diff
@@ -53,6 +53,7 @@ goog.scope(function() {
      * - pageViewMode: Page view mode (singlePage / spread / autoSpread). default: singlePage
      * - zoom: Zoom factor with which pages are displayed. default: 1
      * - fitToScreen: Auto adjust zoom factor to fit the screen. default: false
+     * - defaultPaperSize: Default paper size in mm. Effective when @page size is set to auto. default: undefined (means the windows size is used as paper size).
```

Comment:
It is preferable for me to use `px` unit because previous PR(#334) provided `getPageSizes` API whose unit is `px`.